isso/views/comments.py

```diff
@@ -336,6 +337,20 @@ def new(self, environ, request, uri):
         resp.headers.add("X-Set-Cookie", cookie("isso-%i" % rv["id"]))
         return resp
 
+    def _remote_addr(self, request):
+        '''
```

Comment:
Please use regular docstring conventions (https://www.python.org/dev/peps/pep-0257/), i.e. """ rather than ''' and a first line standalone summary.

Reply:
Ok, improved !